code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -22,17 +22,24 @@
 		var/datum/game_mode/cult/cult = ticker.mode
 		orbs_needed = cult.orbs_needed
 	black_overlay = image('icons/obj/cult_large.dmi', "shell_narsie_black")
+	SSshuttle.emergencyNoEscape = 1
 
 /obj/structure/constructshell/large/Destroy()
 	priority_announce("The extra-dimensional flow has ceased. All personnel should return to their routine activities.","Central Command Higher Dimensions Affairs")
```

Comment:
Shouldn't this not be announced if narsie is summoned? Since this seems like it should tie into the red alert.